### DISCUSSION TITLE
how to start interfacing with two different turbine models?

### DISCUSSION THREAD

**Discussion by clarissa-eccard** [NatLabRockies/floris#250] [Q&A] [answered]
Hi everyone,
I would like to evaluate the production of a wind farm with different turbine models. How can I define two different models for the same wind farm?

Thanks in advance

**Comment by bayc** [comment, marked as answer]
Hello,

The turbine's properties are defined in the input `.json` file, where you would need to specify the Cp (`power`), Ct (`thrust`), and wind speed (`wind_speed`) tables. You can see our `example_input.json` for reference here: https://github.com/NREL/floris/blob/main/examples/example_input.json

You can also change turbine properties programatically, as demonstated in the examples here: https://github.com/NREL/floris/tree/main/examples/change_turbine

The latter would be the most useful for your case, as you can define one turbine in the input `.json` and then update specific turbines to reflect the other turbine type. Using the `change_turbine` function as shown in the examples, you can also update the Cp/Ct/wind speed tables. We currently don't support defining multiple turbines in the `.json`.

**Reply by Bartdoekemeijer** [reply]
Linking feature request #223

**Reply by clarissa-eccard** [reply]
Thanks for the help. Your tip solved my problem and I managed to get good results in my analyses. I used the change_turbine and replaced the Cp/Ct tables and it worked.
